susemanager/src/mgr_bootstrap_data.py

```diff
@@ -1396,61 +822,6 @@
         "PKGLIST": PKGLISTMICRO_BUNDLE_ONLY,
         "DEST": DOCUMENT_ROOT + "/pub/repositories/slemicro/5/5/bootstrap/",
     },
-    "openSUSE-Leap-15-x86_64": {
-        "BASECHANNEL": "opensuse_leap15_0-x86_64",
-        "PKGLIST": PKGLIST15_SALT + PKGLIST15_X86_ARM,
-        "DEST": DOCUMENT_ROOT + "/pub/repositories/opensuse/15/0/bootstrap/",
-    },
-    "openSUSE-Leap-15.1-x86_64-uyuni": {
-        "BASECHANNEL": "opensuse_leap15_1-x86_64",
-        "PKGLIST": PKGLIST15_SALT + PKGLIST15_X86_ARM,
-        "DEST": DOCUMENT_ROOT + "/pub/repositories/opensuse/15/1/bootstrap/",
-    },
-    "openSUSE-Leap-15.1-aarch64-uyuni": {
-        "BASECHANNEL": "opensuse_leap15_1-aarch64",
-        "PKGLIST": PKGLIST15_SALT + PKGLIST15_X86_ARM,
-        "DEST": DOCUMENT_ROOT + "/pub/repositories/opensuse/15/1/bootstrap/",
-    },
-    "openSUSE-Leap-15.1-x86_64": {
-        "PDID": [1929, 1712],
-        "PKGLIST": PKGLIST15_SALT + PKGLIST15_X86_ARM,
-        "DEST": DOCUMENT_ROOT + "/pub/repositories/opensuse/15/1/bootstrap/",
-    },
-    "openSUSE-Leap-15.2-x86_64": {
-        "PDID": [2001, 1712],
-        "PKGLIST": PKGLIST15_SALT + PKGLIST15_X86_ARM,
-        "DEST": DOCUMENT_ROOT + "/pub/repositories/opensuse/15/2/bootstrap/",
-    },
-    "openSUSE-Leap-15.2-x86_64-uyuni": {
-        "BASECHANNEL": "opensuse_leap15_2-x86_64",
-        "PKGLIST": PKGLIST15_SALT + PKGLIST15_X86_ARM,
-        "DEST": DOCUMENT_ROOT + "/pub/repositories/opensuse/15/2/bootstrap/",
-    },
-    "openSUSE-Leap-15.2-aarch64-uyuni": {
-        "BASECHANNEL": "opensuse_leap15_2-aarch64",
-        "PKGLIST": PKGLIST15_SALT + PKGLIST15_X86_ARM,
-        "DEST": DOCUMENT_ROOT + "/pub/repositories/opensuse/15/2/bootstrap/",
-    },
-    "openSUSE-Leap-15.3-x86_64": {
-        "PDID": [2236, 1712],
-        "PKGLIST": PKGLIST15_SALT + PKGLIST15_X86_ARM,
-        "DEST": DOCUMENT_ROOT + "/pub/repositories/opensuse/15/3/bootstrap/",
-    },
-    "openSUSE-Leap-15.3-aarch64": {
-        "PDID": [2233, 1709],
-        "PKGLIST": PKGLIST15_SALT + PKGLIST15_X86_ARM,
-        "DEST": DOCUMENT_ROOT + "/pub/repositories/opensuse/15/3/bootstrap/",
-    },
-    "openSUSE-Leap-15.3-x86_64-uyuni": {
-        "BASECHANNEL": "opensuse_leap15_3-x86_64",
-        "PKGLIST": PKGLIST15_SALT + PKGLIST15_X86_ARM,
-        "DEST": DOCUMENT_ROOT + "/pub/repositories/opensuse/15/3/bootstrap/",
-    },
-    "openSUSE-Leap-15.3-aarch64-uyuni": {
-        "BASECHANNEL": "opensuse_leap15_3-aarch64",
-        "PKGLIST": PKGLIST15_SALT + PKGLIST15_X86_ARM,
-        "DEST": DOCUMENT_ROOT + "/pub/repositories/opensuse/15/3/bootstrap/",
-    },
     "openSUSE-Leap-15.4-x86_64": {
```

Comment:
Leap 15.4 is gone too for one year now

Reply:
I am not sure if it was still used in one of our testsuites.